docs/csi-debug.md

```diff
@@ -53,3 +53,18 @@ dir
 mkdir /tmp/test
 mount -v -t nfs -o vers=4,minorversion=1,sec=sys accountname.blob.core.windows.net:/accountname/filesharename /tmp/test
 ```
+
+
+### Troubleshooting performance issues on Azure Files
+
+##### File shares are being throttled and overall performance is slow 
+Find out whether you have been throttled by looking at your Azure Monitor by following this documentation - [Link](https://docs.microsoft.com/en-us/azure/storage/files/storage-troubleshooting-files-performance#cause-1-share-was-throttled)
+
+###### Standard Files
+
+Enable [larger file shares](https://docs.microsoft.com/azure/storage/files/storage-files-how-to-create-large-file-share?tabs=azure-portal) on your storage account. Large file shares support up to 10,000 IOPS per share at no extra cost on non-premium tier. Right now, Large File Shares need to be manually turned on the storage account and quota needs to be set to 100 TiB to get 10K IOPS. In general - one does not need remounting of the file share after the setting is changed, however it is recommended to turn it on at Storage Account creation time. Same storage account can have multiple LargeFileShare in Standard Storage. We are looking into providing it as an option on the CSI driver in future. 
```

Comment:
Rephrase "In general - one does not need remounting of the file share after the setting is changed, however it is recommended to turn it on at Storage Account creation time." to "We recommend that you enable this when creating the Storage Account. However, if you do enable this at a later time, you will not need to remount the file share. "